docs/TranslatableFields.md

```diff
@@ -28,9 +36,16 @@ This is how to use it:
 <TranslatableFields locales={['en', 'fr']}>
```

Comment:
Could you make this example a bit more "executable", i.e. insert it in a `<Show>` view and add the import statement?